packages/tailwindcss/src/candidate.ts

```diff
@@ -255,21 +252,13 @@ export function* parseCandidate(input: string, designSystem: DesignSystem): Iter
     base = base.slice(1)
   }
 
-  // Candidates that start with a dash are the negative versions of another
-  // candidate, e.g. `-mx-4`.
-  if (base[0] === '-') {
-    negative = true
-    base = base.slice(1)
-  }
```

Comment:
Wondering if there are any migrations in the upgrade script that will no longer work now because we compare the `base` ignoring a potential `-`. I looked over `simple-legacy-classes` and it looks fine though. `max-w-screen-*` looked sus but didn't support negative prefixes in v3 either.